src/plugins/intel_gpu/src/plugin/transformations/bcast_and_pad_zp_buffers.cpp

```diff
@@ -106,8 +143,17 @@ BroadcastAndPadZeroPointBuffers::BroadcastAndPadZeroPointBuffers(size_t pad_size
 
         if (auto wzp = std::dynamic_pointer_cast<ov::op::v0::Constant>(pattern_map.at(wzp_m).get_node_shared_ptr())) {
             auto target_shape = wzp->get_shape();
-            const size_t wzp_c_idx = 0;
-            auto aligned_wzp = pad_quantization_parameter(wzp, out_shape[channel_idx].get_length(), wzp_c_idx, pad_size);
+
+            std::shared_ptr<ov::Node> aligned_wzp;
+            if (supports_immad && !(conv->get_groups() > 1)) {
```

Comment:
could you add a comment here?
`// OneDNN supports scalar wzp. If wzp data are identical, replace it with scalar value for OneDNN`

Reply:
Added